d2l-icon.html

```diff
@@ -64,7 +63,32 @@
 					}
 				}
 				return false;
+			},
+
+			__getIronIcon: function() {
+				return this.$$('iron-icon');
+			},
+
+			__initialize: function() {
+				var svg = this.__findSvg(this);
+				if (!svg) {
+					return false;
+				}
+				/* IE prevent svg from being focusable */
+				svg.setAttribute('focusable', 'false');
+				var mirrorRtl = this.__computeMirrorRtl(svg);
+				if (mirrorRtl) {
+					this.$$('iron-icon').setAttribute('data-mirror-rtl', '');
```

Comment:
Ideally I would not need such a helper.  A perceptual diff would enable testing the flip case better.  :)